src/Extension/Footnote/Renderer/FootnoteBackrefRenderer.php

```diff
@@ -43,7 +43,7 @@ public function render(Node $node, ChildNodeRendererInterface $childRenderer)
         $attrs['href']  = \mb_strtolower($node->getReference()->getDestination());
         $attrs['role']  = 'doc-backlink';
 
-        return '&nbsp;' . new HtmlElement('a', $attrs, '&#8617;', true);
+        return '&nbsp;' . new HtmlElement('a', $attrs, $this->config->get('footnote/backref_text', '&#8617;'), true);
```

Comment:
Thanks, @bakerkretzmar! This definitely should be a configuration option; probably just got overlooked when porting it from the original project (which had none).

One tiny note:

As as done in #506 (https://github.com/thephpleague/commonmark/pull/506/files#diff-5b5681fa1e8206c321de09d716ed6533), this needs to be escaped using `htmlspecialchars`. It should also probably be renamed to `backref_symbol`. If someone needs more complex styling, they could remove this symbol entirely and style it by targeting it via CSS using `backref_class`.

Reply:
Thanks, updated!